### PR TITLE
1201 collect all event logs on first run

### PR DESCRIPTION
**Pull request by souravbaner-da**
For first run in overwatch cluster_events_bronze would now get all the records instead of some specific timestamp

### CODE REVIEW

**Review by sriram251-code — APPROVED** [review]

**Comment by neilbest-db** [comment]
@souravbaner-da, I believe this is the one where you agreed to adjust the method signature and/or argument types for `landClusterEvents` in order to pass the code analysis check, correct?  You may request my review when that is ready.  Thanks.

File: src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala
Comment (neilbest-db) [inline src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala:1]: I added some whitespace here for legibility.

**Review by neilbest-db — APPROVED** [review]
LGTM.

**Comment by neilbest-db** [comment]
This is ready to be merged but I am not yet able to do that.  I moved it to "Done" on the board but the PR icon there will remind us that it's open.

**Pull request merged**